packages/main/src/themes/sap_horizon/DayPicker-parameters.css

```diff
@@ -36,6 +36,14 @@
 	--_ui5_daypicker_item_weeekend_filter: brightness(105%);
 	--_ui5_daypicker_item_selected_hover: var(--sapList_Hover_Background);
 	--_ui5_daypicker_item_now_inset: 0.375rem;
-	--_ui5-dp-item_withsecondtype_border: 0.1875rem;
+	--_ui5-dp-item_withsecondtype_border: 0.25rem;
+	--_ui5_daypicker_item_selected__secondary_type_text_outline: 0.125rem solid var(--sapSelectedColor);
+	--_ui5_daypicker_two_calendar_item_now_day_text_contetn: "";
+	--_ui5_daypicker_two_calendar_item_now_selected_border_width:  0.1875rem;
+	--_ui5_daypicker_two_calendar_item__border_redius: 0.5rem;
```

Comment:
item__border -> item_border - just one underscore is enough

Reply:
done